frontend/src/pages/projects/screens/spawner/SpawnerPage.tsx

```diff
@@ -30,12 +30,18 @@ import useNotebookImageData from '../detail/notebooks/useNotebookImageData';
 import useNotebookDeploymentSize from '../detail/notebooks/useNotebookDeploymentSize';
 import { getRootVolumeName, useMergeDefaultPVCName } from './spawnerUtils';
 import { useNotebookEnvVariables } from './environmentVariables/useNotebookEnvVariables';
+import { AppContext } from '../../../../app/AppContext';
 
 type SpawnerPageProps = {
   existingNotebook?: NotebookKind;
 };
 
 const SpawnerPage: React.FC<SpawnerPageProps> = ({ existingNotebook }) => {
+  const {
+    dashboardConfig: {
+      spec: { notebookController },
+    },
+  } = React.useContext(AppContext);
```

Comment:
I would have expected this in the footer -- as it has nothing to do with the spawner page. Please move it so we can keep this component as thin as we can -- it has a tendency to become mammoth.